File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        int arr[] = new int [nums.length];
        ArrayList<Integer> numbers = new ArrayList<>();
        for (int x : nums){
            arr[x-1]++;
        }
        for (int i = 0 ;i<nums.length;i++){
            if ( arr[i] == 0 ) numbers.add(i+1);
        }
        return numbers;
    }
}

File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        vector<int> arr(nums.size(), 0);
        vector<int> numbers;

        for (int x : nums) {
            arr[x - 1]++;
        }

        for (int i = 0; i < nums.size(); i++) {
            if (arr[i] == 0) {
                numbers.push_back(i + 1);
            }
        }

        return numbers;
    }
};